config/file.go

```diff
@@ -27,11 +25,10 @@ var (
 //
 // It also uses the folder containing the configuration file for storing other configuration files.
 type FileStore struct {
```

Comment:
We should document in the file and memory stores that these backing stores are not goroutine safe anymore, and they should only be accessed through the upper store, or synchronize calls themselves.